src/se/vidstige/jadb/AdbFilterInputStream.java

```diff
@@ -1,12 +1,21 @@
 package se.vidstige.jadb;
 
+import java.io.BufferedInputStream;
 import java.io.FilterInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
 public class AdbFilterInputStream extends FilterInputStream {
-    public AdbFilterInputStream(InputStream inputStream) {
-        super(inputStream);
+
+    private Transport mTransport;
```

Comment:
What's the point of this? The thought here is that this stream class just works on a `InputStream` without know what kind of stream that is.